projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts

```diff
@@ -1,168 +1,423 @@
+            });
+
+            it('should handle invalid props', async () => {
+                element.setProperty('value', 'key|value,key2|value2');
+                element.setAttribute('list-delete-label', []);
+                await page.waitForChanges();
+
+                const list = await getList();
+                expect(list.getAttribute('button-label')).toBe('');
+            });
+        });
+
+        describe('disabled', () => {
+            it('should set disabled to child', async () => {
+                element.setProperty('disabled', true);
+                await page.waitForChanges();
+
+                const form = await getForm();
+                const list = await getList();
+
+                expect(form.getAttribute('disabled')).toBeDefined();
+                expect(list.getAttribute('disabled')).toBeDefined();
+            });
+
+            it('should not set disabled to child', async () => {
+                const form = await getForm();
+                const list = await getList();
+
+                expect(form.getAttribute('disabled')).toBeNull();
+                expect(list.getAttribute('disabled')).toBeNull();
+            });
+        });
+
+        describe('hint', () => {
+            it('should render', async () => {
+                element.setProperty('hint', 'Some hint');
+                await page.waitForChanges();
+
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint.innerText).toBe('Some hint');
+                expect(hint.getAttribute('id')).toBe('hint-some-hint');
+            });
+
+            it('should not render', async () => {
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint).toBeNull();
+            });
+
+            it('should handle invalid', async () => {
+                element.setProperty('hint', { a: 'object' });
+                await page.waitForChanges();
+
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint).toBeNull();
+            });
+        });
+
+        describe('label', () => {
+            it('should render', async () => {
+                element.setProperty('label', 'Some label');
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('Some label');
+            });
+
+            it('should not render', async () => {
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('');
+            });
+
+            it('should handle invalid', async () => {
+                element.setProperty('label', ['some', 'array']);
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('some,array');
+            });
+        });
```

Comment:
should we have test for uindefined and null too? (I mean for all props)

Reply:
At least one of those, because is an attr.